orchestra/static/dist/main.js

```diff
@@ -47392,17 +47392,17 @@ function orchestraApi($http) {
       });
```

Comment:
QA
- [ ] rows appear in the same order as they do in project management
- [ ] no staffing buttons/input fields/show-hide buttons appear for experts
- [ ] all staffing buttons/input fields/show-hide buttons appear for experts